src/components/RegistrationForm.tsx

```diff
@@ -249,7 +249,11 @@ export const RegistrationForm = ({ data }) => {
                     <Modal.Header closeButton>
                         <Modal.Title>Algorithm submitted for registration</Modal.Title>
                     </Modal.Header>
-                    <Modal.Body>Your algorithm was submitted for registration. You can view the progress here: <a href={registrationUrl} target="_blank">{registrationUrl}</a></Modal.Body>
+                    <Modal.Body>
+                        Your algorithm was submitted for registration. You can view the progress here: <a href={registrationUrl} target="_blank">{registrationUrl}</a>
+                        <br />
```

Comment:
Adding an extra break to make the text more readable.

```suggestion
                        <br />
                        <br />
```